boards/st/b_u585i_iot02a/b_u585i_iot02a.dts

```diff
@@ -112,3 +112,15 @@
 	st,adc-prescaler = <4>;
 	status = "okay";
 };
+
+&mx25lm51245 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			reg = <0x00000000 DT_SIZE_M(64)>;
+		};
+	};
+};
```

Comment:
This is already set at lines 79..89.
It was brought by 49ed8318c296217b13d320d97d72f0f99f657d41.